recipes/iceoryx/all/test_package/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# Build icehello example
+
+cmake_minimum_required(VERSION 3.5)
+project(examples)
+include(${CMAKE_BINARY_DIR}/conanbuildinfo.cmake)
+conan_basic_setup()
+
```

Comment:
Regarding this comment: https://github.com/conan-io/conan-center-index/pull/5541#issuecomment-842865998

If we only want to test linkage then only these examples are sufficient:
- icedelivery
- icedelivery_in_c
- singleprocess for roudi daemon build

All other example code can be removed.